.github/workflows/release.yml

```diff
@@ -8,6 +8,9 @@ on:
         required: false
         type: boolean
         default: true
+    push:
+    branches:
```

Comment:
be careful to the indentation.
branches: needs to be indented

Reply:
Updated in new commit. Hope it has been submitted.

Reply:
you can check ;) anyway you can merge